contracts/interfaces/IGovernorModule.sol

```diff
@@ -30,16 +31,16 @@ interface IGovernorModule {
 
     // The following functions are overrides required by Solidity.
 
-    enum ProposalState {
-        Pending,
-        Active,
-        Canceled,
-        Defeated,
-        Succeeded,
-        Queued,
-        Expired,
-        Executed
-    }
+    // enum ProposalState {
+    //     Pending,
+    //     Active,
+    //     Canceled,
+    //     Defeated,
+    //     Succeeded,
+    //     Queued,
+    //     Expired,
+    //     Executed
+    // }
```

Comment:
What's going on here? Why comment this enum out?

Reply:
Commented out to test that all the tests ran, and forgot to delete it. Fixed that in recent commit.

That declared enum was a duplicate of an enum declared down in the OZ contracts, which we need to use. Otherwise the compiler throws an error that the multiple functions that are being overriden by the state function return different types. They were all return "ProposalState" enums, but were pointing to different instances of them in the code.